### PR TITLE
fixes #7430 - Adds subnet mask validation

### PR DESCRIPTION
**Pull request by cfouant**

### CODE REVIEW

File: test/unit/subnet_test.rb [hunk 1]
```diff
@@ -40,6 +40,12 @@ def setup
     assert !@subnet.save
   end
 
+  test "mask should have valid address" do
+    @subnet.mask = "255.0.0.255"
+    set_attr(:network=, :domains=, :name=)
+    assert !@subnet.save
```
Comment (dLobatog) [inline test/unit/subnet_test.rb]: Please use `refute` instead of `assert !`

File: lib/net/validations.rb [hunk 2]
```diff
@@ -30,6 +31,12 @@ def validate_ip ip
       ip
     end
 
+    #validates the subnet mask
```
Comment (dLobatog) [inline lib/net/validations.rb]: This comment doesn't really say anything that the method name doesn't say already :smile:

**Comment by dLobatog** [comment]
Works well, ready to merge after fixing the two minor quirks.

**Comment by cfouant** [comment]
Changes submitted, thanks!

**Comment by ohadlevy** [comment]
merged as 7af45f0 thanks @cfouant !

**Pull request closed without merging**